main.py

```diff
@@ -226,6 +231,27 @@ def string2timestamp(s):
 def year2timestamp(y):
     return time.mktime(datetime.date(y, 1, 1).timetuple())
 
+@app.route('/newFeatures', methods=['POST'])
+def updatebyemail():
+    emailaddress = str(request.json['address'])
+    if len(emailaddress)>40:
+        return jsonify(respo='Email too long', emailaddress = emailaddress)
+    try:
+        con = sqlite3.connect('emailAddresses.db')
```

Comment:
I need to save in the database only the email address and in the users database there are other details as well. About the FLASK ORM - the reason  I used pure sqlite is that I am not familiar with SQLAlchemy. I will read about SQLAlchemy and try to implement it with it.